README.md

```diff
@@ -1,12 +1,18 @@
 # News Site
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+I created a news site about guinea pigs called the Wheeeeekly News. I wanted a colorful, fun, and responsive site just like my favorite pets. 
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+My biggest problem was the header and understanding media queries and getting my website to properly function with them at breakpoints. 
+
+It seemed no matter what I did the site wouldn't respond to any of the code I was writing. I googled, rewatching tutorials, and eventually talked it out with one of my teammates. My team mate was able to explain media queries to me that triggered a solution in my brain. I realized that I had originally designed my website in desktop view, which is why trying to apply mobile media queries first wasn't working. The heirarchies were all wrong!
+
+I went back and adjusted my css (before media queries) to mobile view and then began applying the queries from there (tablet -> desktop) and it worked! I was able to push forward after that and complete my site.
```

Comment:
Great to hear that you managed to find an alternative approach and pursue it to your benefit already!  Well done